### ISSUE TITLE
关系提取例子中计算 Loss 的疑问

### ISSUE THREAD

**Issue by moriwang** [bojone/bert4keras#240]
苏神你好，我对于下面这段代码有些疑问，查看 Loss Layer 的实现后还是有几点不明白。

1. TotalLoss 中的 mask[4] 是指什么呢？ call 的时候需要传入 input 和 mask，是否最后一段计算的时候没有传入 mask？
2. [2,3] 指的是 output_axis 吗？为何是输出一个向量而不是一个数值呢？ 

还有一点疑问是关于预训练模型的，以 bert 来说，输入需要是 token, position, segment 三种，范例中只有 token 和 segment，是否 bert4keras 自动做了 position embedding，用来消除 sequence_padding 时添加的 0 ？

### 核心代码
```python
class TotalLoss(Loss):
    """subject_loss与object_loss之和，都是二分类交叉熵
    """
    def compute_loss(self, inputs, mask=None):
        subject_labels, object_labels = inputs[:2]
        subject_preds, object_preds, _ = inputs[2:]
        if mask[4] is None:
            mask = 1.0
        else:
            mask = K.cast(mask[4], K.floatx())
        # sujuect部分loss
        subject_loss = K.binary_crossentropy(subject_labels, subject_preds)
        subject_loss = K.mean(subject_loss, 2)
        subject_loss = K.sum(subject_loss * mask) / K.sum(mask)
        # object部分loss
        object_loss = K.binary_crossentropy(object_labels, object_preds)
        object_loss = K.sum(K.mean(object_loss, 3), 2)
        object_loss = K.sum(object_loss * mask) / K.sum(mask)
        # 总的loss
        return subject_loss + object_loss


subject_preds, object_preds = TotalLoss([2, 3])([
    subject_labels, object_labels, subject_preds, object_preds,
    bert.model.output
])
```

**Comment by bojone** [comment]
1. mask[4]就是第4个输入对应的mask矩阵，这是keras自带的mask机制；
2. [2,3]指的是该层直接返回第2个和第3个输入，Loss层仅仅是为了添加loss，不做任何操作，直接返回（部分）原始输出。

**Comment by moriwang** [comment]
原来如此，我再去读一下 mask 部分的源码，谢谢苏神解惑。
